src/ExceptionAll.Client/Clients/ExceptionAllClient.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace ExceptionAll.Client.Clients;
+
+/// <summary>
+/// <inheritdoc cref="IExceptionAllClient"/>
+/// </summary>
+public class ExceptionAllClient : IExceptionAllClient
+{
+    private readonly HttpClient _httpClient;
+    private readonly ISerializerService _serializer;
+    private readonly IValidationService _validation;
+    private readonly JsonSerializerOptions? _options;
+
+    public ExceptionAllClient(
+        HttpClient httpClient,
+        ISerializerService serializer,
+        IValidationService validation,
+        JsonSerializerOptions? options = null)
+    {
+        _httpClient = httpClient;
+        _serializer = serializer;
+        _validation = validation;
+        _options    = options;
+    }
+
+    ///<inheritdoc/>
+    /// <exception cref="HttpRequestException"/>
+    /// <exception cref="TaskCanceledException"/>
+    /// <exception cref="InvalidOperationException"/>
+    public async ValueTask<T?> GetContentAsync<T>(string relativeUrl, CancellationToken token = new())
```

Comment:
You should default the CancellationToken to `default` instead of initializing a new one that may not ever be used (if it's not being passed in, it probably won't ever get cancelled, ya know?)